feat(table): Table component

justwiner opened the pull request:
<!-- Specify your pull request type -->
- [ ] Component Migration

justwiner committed:
feat(table): upload table source code(vue2)

Commit:
feat(table): upload table by ts

justwiner committed:
feat(table): 10%

Commit:
feat: update

Commit:
feat: update

Commit:
feat: upload

zazzaz commented on packages/utils/util.ts:
```diff
@@ -156,3 +156,18 @@ export function getRandomInt(max: number) {
 
 export { isVNode } from 'vue'
+
+// TODO: use native Array.find, Array.findIndex when IE support is dropped
```
Just use Array.find, please

justwiner replied:
Ok!
Optimization will be carried out later, including type checking

zazzaz commented:
Cool~

@justwiner Just a reminder that it's better to rebase the origin/dev branch daily to reduce conflicts.

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): 30%

justwiner committed:
feat(table): 31%

justwiner committed:
feat(table): 35%

justwiner committed:
feat: add lint fix

justwiner committed:
feat(table): 45%

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): 50%

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): 70%

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

zazzaz commented on packages/table/src/store/helper.ts:
```diff
@@ -1,5 +1,5 @@
 import Store from './index'
-import debounce from 'throttle-debounce/debounce'
```
please use `lodash` instead, package throttle-debounce will be removed later

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): 85%

Most of the functions have been completed, and there are still multi-level headers and tooltips that
have not been implemented. The code of the table component will be optimized later

justwiner committed:
feat(table): complete tooltip

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): optimized code

justwiner committed:
feat(table): 95%

justwiner committed:
feat(table): add unit test

justwiner committed:
feat(table): add unit test

justwiner committed:
feat(table): complete the test case

justwiner committed:
Merge remote-tracking branch upstream/dev into dev

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
feat(table): 98%

justwiner committed:
docs(table): table.md: Modify the template syntax of vue3

justwiner committed:
Merge remote-tracking branch 'upstream/dev' into dev

justwiner committed:
docs(table): update table.md

justwiner committed:
feat(table): complete multi level header

justwiner committed:
Merge pull request #11 from justwiner/feat/mulHeader

Feat/mul header
